apps/dashboard/src/app/(dashboard)/(chain)/components/server/products.ts

```diff
@@ -37,6 +37,11 @@ export const products = [
     id: "pay",
     icon: PayIcon,
   },
+  {
+    name: "Faucet",
+    id: "faucet",
+    icon: PayIcon,
```

Comment:
has to get its own icon, especially since this is right next to pay and has the same icon

Reply:
also this should be added dynamically on testnets only probably (aka don't hard code it into this list for every chain (I think))

Reply:
removed the faucet from the list for now